File: app/src/main/java/me/codeminions/mvpproject/mvp/view/ILoginView.java
package me.codeminions.mvpproject.mvp.view;

public interface ILoginView {
    String getUserName();
    String getUserPwd();

    void showLoading();
    void hideLoading();

    void loginSuccess();
    void loginFail();

    void cleanUserName();
    void cleanUserPwd();
}

File: app/src/main/java/me/codeminions/mvpproject/kotmvp/loginMVP/view/ILoginView.kt
package me.codeminions.mvpproject.kotmvp.loginMVP.view

import me.codeminions.mvpproject.kotmvp.IView
import me.codeminions.common.bean.User

interface ILoginView : IView {
    fun onLoginSuccess(user: User)
    fun onLoginFail(error: String)

    fun getUserName(): String
    fun getUserPwd(): String

    fun cleanUserName()
    fun cleanUserPwd()

    fun showLogining()
    fun hideLogining()
}